pkg/connector/backfill.go

```diff
@@ -24,10 +24,6 @@ func (t *TelegramClient) getTakeoutID(ctx context.Context) (takeoutID int64, err
 		return t.userLogin.Metadata.(*UserLoginMetadata).TakeoutID, nil
 	}
 
-	if t.takeoutAccepted == nil {
-		t.takeoutAccepted = make(chan struct{})
-	}
-
 	for {
 		accountTakeout, err := t.client.API().AccountInitTakeoutSession(ctx, &tg.AccountInitTakeoutSessionRequest{
```

Comment:
the event should probably still get `.Clear()`ed somewhere here to prevent infinite loops (either before the loop or after the error)